browser/data-browser/src/views/TablePage/helpers/formatNumber.ts

```diff
@@ -21,6 +22,22 @@ export function formatNumber(
     return formatter.format(value / 100);
   }
 
+  if (formatting === urls.instances.numberFormats.currency) {
+    try {
+      const formatter = new Intl.NumberFormat('default', {
+        style: 'currency',
+        currency,
+        currencyDisplay: 'narrowSymbol',
+      });
+
+      return formatter.format(value);
+    } catch (e) {
+      console.error(e);
```

Comment:
This should be `handleError`

Reply:
Oh this catch should actually do nothing and just let the formatter continue to format the value as if it were just a number as this only triggers when the currency is not supported by the browser.